src/main/java/org/terracotta/statistics/ZeroOperationStatistic.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * All content copyright Terracotta, Inc., unless otherwise indicated.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics;
+
+import org.terracotta.statistics.observer.ChainedOperationObserver;
+
+import java.util.Set;
+
+/**
+ * An operation statistic that always return 0 for everything. Used as a null object.
+ */
+public class ZeroOperationStatistic<T extends Enum<T>> implements OperationStatistic<T> {
+
+  @SuppressWarnings("rawtypes")
+  private static final OperationStatistic<?> INSTANCE = new ZeroOperationStatistic();
+
+  @SuppressWarnings("unchecked")
+  public static <T extends Enum<T>> OperationStatistic<T> get() {
+    return (OperationStatistic<T>) INSTANCE;
+  }
+
+  @Override
+  public Class<T> type() {
+    return null;
+  }
+
+  @Override
+  public ValueStatistic<Long> statistic(T result) {
+    return null;
```

Comment:
Shouldn't we return a ValueStatistic returning 0?

Reply:
0 or -1 are good values for statistics. In the management layer, we are discarding every statistic that have a null value (meaning, the stat is unavailable or cannot be computed or something else).